docs/source/getting_started.rst

```diff
@@ -1,2 +1,155 @@
+orchestra, take a look at our :doc:`API documentation <api>`.
+
+
+********************
+Install Dependencies
+********************
+
+Orchestra requires python 3 and Django version 1.8 or higher to run, so make
+sure you
+`have them installed <https://docs.djangoproject.com/en/1.8/topics/install/>`_.
+We recommend setting up a
+`virtual environment <http://docs.python-guide.org/en/latest/dev/virtualenvs/>`_
+to isolate your python dependencies, and we're fond of
+`virtualenvwrapper <https://virtualenvwrapper.readthedocs.org/en/latest/>`_ to
+make that process easier.
+
+Orchestra requires a number of python dependencies to run. You can install them
+by simply pulling down and installing our ``requirements.txt`` file::
+
+  wget https://raw.githubusercontent.com/unlimitedlabs/orchestra/v0.1.0/requirements.txt
+  pip install -r requirements.txt
+
+
+***********************
+Create a Django Project
+***********************
+
+Orchestra is a Django app, which means that it must be run within a Django
+project (read up at `the tutorial
+<https://docs.djangoproject.com/en/1.8/intro/tutorial01/#creating-a-project>`_
+if that's confusing). Start a project with
+``django-admin startproject your_project``, replacing ``your_project`` with
+your favorite project name. From here on out, this document will assume that
+you stuck with ``your_project``, and you should replace it appropriately.
+
+
+*******************************
+Install and Configure Orchestra
+*******************************
+
+Next, let's get Orchestra installed and running. To get the code, just install
+using pip: ``pip install orchestra``.
+
+Orchestra has a number of custom settings that require configuration before
+use. First, download the default orchestra settings file and place it next to
+the project settings file::
+
+  wget https://raw.githubusercontent.com/unlimitedlabs/orchestra/v0.1.0/example_project/example_project/orchestra_settings.py
+  mv orchestra_settings.py your_project/your_project
+
+Next, edit the ``orchestra_settings.py`` file:
+
+* Add ``'simple_workflow'`` to ``settings.INSTALLED_APPS`` in the "General"
+  section if you want to run the demo workflow
+  (:ref:`instructions below <demo-section>`).
+
+* Change settings like the ``ORCHESTRA_PROJECT_API_SECRET`` from ``'CHANGEME'``
+  to more appropriate values.
+
+* Add 3rd party credentials in the "3rd Party Integrations" section so that
+  Orchestra can store files on `Amazon s3 <https://aws.amazon.com/s3/>`_ and use
+  `Google apps <http://apps.google.com>`_ and `Slack <https://slack.com/>`_ to
+  help communicate with expert workers (though these are all optional).
+
+Then, at the bottom of your existing settings file
+(``your_project/your_project/settings.py``), import the orchestra
+settings::
+
+  from .orchestra_settings import setup_orchestra
+  setup_orchestra(__name__)
+
+You'll also need to set up orchestra's URLs, so that Django knows where to
```

Comment:
Clarify? The bulleted list is about changes to make to `orchestra_settings.py`. This paragraph is about changes to add to `urls.py`. Are you talking about the `ORCHESTRA_URL` setting, which defaults to `127.0.0.1:8000`? I don't mention it here because it should just work with a default setup, but can add it in if you think that's important.